jabgui/src/main/java/org/jabref/cli/GuiCommandLine.java

```diff
@@ -16,6 +16,13 @@ public class GuiCommandLine {
     @Option(names = {"-a", "--add"}, description = "Add to currently opened library.")
     public boolean append;
 
+    @Option(names = {"--importBibTex"}, description = "Import bibtex string")
```

Comment:
I think, this is not used by the exntesion, is it?

If yes: Please add Java coment

In all cases: `@Deprecated` :)

Reply:
Used by mac/linux see jabrefhost.py